lib/Serverless.js

```diff
@@ -427,6 +426,16 @@ class Serverless {
       }
     }
 
+    if (
+      process.argv.slice(2).includes('-v') &&
+      _.get(resolveCliInput().commandSchema, 'options.verbose.shortcut') === 'v'
+    ) {
+      this._logDeprecation(
+        'CLI_VERBOSE_OPTION_ALIAS',
+        'Starting with `v3.0.0`, `-v` will no longer be supported as alias for `--verbose` option. Please use `--verbose` flag instead.'
```

Comment:
Our convention is to use `"` as quotes in CLI messages (in this context they look more readable)

Reply:
changed :+1: